config.py

```diff
@@ -6,3 +6,10 @@
 DEBUG = os.environ.get('DEBUG', 0)
 SQLALCHEMY_DATABASE_URI = os.environ['DATABASE_URL']
 SQLALCHEMY_MIGRATE_REPO = os.path.join(basedir, 'db_repository')
+
+BUTTON_ACTIONS = [
```

Comment:
This should be more consistent. Either say "Too slow, too fast, too easy, too hard", or be more verbose and say "this is too _something_".